### PR TITLE
Add Github workflows to automatically build docker images

### PR DESCRIPTION
**Pull request by Oscariremma**
Add a github workflow to automatically build docker images and push them to ghcr

### CODE REVIEW

File: .github/workflows/web-backend.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build Docker image (and push on master)
+        uses: docker/build-push-action@v4.0.0
+        with:
+          push: ${{ github.ref == 'refs/heads/master' }}
+          tags: ghcr.io/cthit/goldapps-web-backend:latest
```
Comment (molleer) [inline .github/workflows/web-backend.yml:30]: Dont you want to push to docker.chalmers.it?

**Review by molleer — COMMENTED** [review]
Could you do the same for ./Dockerfile? It is just a container which allows you to sync accounts via the cli tool.

File: .github/workflows/web-backend.yml (hunk 1 above)
Comment (Oscariremma) [reply .github/workflows/web-backend.yml:30]: Under our year as digIT we have primarily used ghcr as it is easier to use within GitHub Workflows and the images produced are listed in the repo next to the code.

**Review by Supergamer1337 — APPROVED** [review]
lgtm

**Review by molleer — APPROVED** [review]
Lets Gamble, Try Merge

**Pull request merged**